src/sentry/web/client_config.py

```diff
@@ -130,14 +146,26 @@ def _resolve_last_org(session, user, org_context=None):
                 include_teams=False,
             )
 
-    if org_context and org_context.member:
+    has_org_access = bool(org_context and org_context.member)
+
+    if not has_org_access and user_is_authenticated:
+        has_staff = features.has("auth:enterprise-staff-cookie", actor=user)
+        is_active_staff = has_staff and staff.is_active_staff(request)
+        is_active_superuser = not has_staff and superuser.is_active_superuser(request)
+        has_org_access = is_active_staff or is_active_superuser
```

Comment:
@schew2381 just to double check, do we need to check for staff here? superuser should be the only ones that can access customer orgs. staff should be able to hit a few endpoints that are also present in _admin

tl;dr i don't think this is necessary for staff -- we can just check for superuser and propogate the active org

Reply:
I think I'm lacking context, do staff need to have the active org on the session to access _admin? I'm not quite sure what "propagating the accessed org in links" means

Reply:
_admin is not tied to any org, which is why i think we don't have to check for staff here

Reply:
Hmm that makes sense, I think you're right then

Reply:
@dashed I think you can delete the staff part and just include a superuser check